javaparser-core-testing/javaparser-core-testing.iml

```diff
@@ -10,6 +10,11 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.jbehave:jbehave-core:4.3" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: commons-io:commons-io:2.6" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.apache.commons:commons-lang3:3.7" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.apache.commons:commons-text:1.2" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: com.github.valfirst:jbehave-junit-runner:2.1.3" level="project" />
```

Comment:
This looks odd.

Reply:
I have no idea what this exactly means. I guess this was changed by one of the maven tasks. And as a heads up, there are two .iml files in the javaparser-core-testing module, `javaparser-core-testing.iml` and `javaparser-testing.iml`

Reply:
Hmmm, I think I'll just remove these files from the repo again. There's nothing but nonsensical merge problems with them.

Reply:
Just FYI, the presence of two .iml files in the javaparser-core-testing was caused by merging the symbol solver into JavaParser. Back then, the file was called `javaparser-testing.iml`, now it is `javaparser-core-testing.iml`. In other words, `javaparser-testing.iml` is a historical obsolete artifact that can be removed. :-)

Reply:
Correct :-) And if these thing had been on the ignore list, this mistake wouldn't have propagated over the world ;-)